### PR TITLE
Release prep for v0.2

### PR DESCRIPTION
**Pull request by dvdplm**

### CODE REVIEW

**Review by maciejhirsz — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,3 +5,13 @@ The format is based on [Keep a Changelog].
...
 ## [Unreleased]
+
+## [v0.2.0] – 2021-06-04
+[changed] The crate structure changed to several smaller crates, enabling users to pick and choose. The `jsonrpsee` crate works as a façade crate for users to pick&chose what components they wish to use.
+[changed] Starting with this release, the project is assuming `tokio` is the async executor.
+[changed] Revamped RPC subscription/method definition: users now provide closures when initializing the server and it is no longer possible to register new methods after the server started.
+[changed] Refactored the internals from the ground up.
+[added] Support for async methods
+[added] Support for batch requests (http/ws)
+
```
Comment (niklasad1) [inline CHANGELOG.md:16]:
```suggestion
[changed] `proc macros API generation` is only supported for the client implementations
```

**Review by niklasad1 — APPROVED** [review]

**Comment by popzxc** [comment]
Yay! By the way, I'm still willing on helping with the project. Are there any more or less important issues I can pick up?

**Comment by dvdplm** [comment]
@popzxc One thing I've been wanting to look at is the benchmark suite: I have not checked for regressions over the many recent refactorings and I think we should extend the performance tests to include subscriptions and async methods. Any interest in taking that on?

**Comment by popzxc** [comment]
Yeah, sure, will do.

**Pull request merged**